### PR TITLE
Milestone4: Feature/library basic UI

### PR DESCRIPTION
**Pull request by brainconnect93**
- [ ]  Created the App.rb file and pass the method to:
        -   List all books.
        - List all people.
        - Create a person (teacher or student, not a plain Person).
        - Create a book.
        - Create a rental.
        - List all rentals for a given person id. 

- [ ]  Then I created the main.rb file and pass the method that performs the following:
       - Present the user with a list of options to perform.
       - Lets users choose an option.
       - If needed, ask for parameters for the option.
       - Have a way to quit the app.

> Kindly review and approve. Thanks.

## comment on previous review
main.rb:15:1: C: Metrics/CyclomaticComplexity: Cyclomatic complexity for functional is too high. [8/7]
def functional(app)

after i rectified the issue you raised, the linters flagged the input as too high

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+def options
+  print "\nPlease choose an option by entering a number \n\n"
+  print "1 - List all books \n"
+  print "2 - List all people \n"
+  print "3 - Create a person (teacher or student) \n"
+  print "4 - Create a book \n"
+  print "5 - Create a rental \n"
+  print "6 - List all rentals for a person \n"
+  print "7 - Quit \n"
```
Comment (Ismailco) [inline main.rb]: - [x] The app closes if the user clicks `enter` two times, kindly fix this so the user can exit the app only when selecting option `7`

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    print 'Book Title:'
+    title = gets.chomp
+    print 'Book Author:'
+    author = gets.chomp
+    new_book = Book.new(title, author)
+    @books << new_book
+    puts "#{new_book.title} created successfully"
+  end
+
+  def create_rental
```
Comment (Ismailco) [inline app.rb:65]: - [ ] When creating the rental an error produced, kindly fix it so the user can creat a rental with user id

Screenshot_20220909_191617

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @brainconnect93, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 2 above)
Comment (brainconnect93) [reply app.rb:65]: rectified. Thanks

File: main.rb (hunk 1 above)
Comment (brainconnect93) [reply main.rb]: main.rb:15:1: C: Metrics/CyclomaticComplexity: Cyclomatic complexity for functional is too high. [8/7]
def functional(app)

after i rectified the issue you raised, the linters flagged the input as too high

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @brainconnect93 , 
Good job implementing some of the changes requested by the previous reviewer.

## Required Changes ♻️
- [ ] Please kindly fix the suggestion by the previous reviewer. At the moment, when we enter is clicked the app exits.

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: main.rb [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+#     functional(app)
+#   end
+# end
+
+# def main
+#   app = App.new
+#   functional(app)
+# end
+
+# main
```
Comment (CollinsTatang) [inline main.rb]: ### **Optional**
- [x] Please remove all commented code from your codebase to make your code clean and easy to maintain.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 3 above)
Comment (brainconnect93) [reply main.rb]: done as requested

**Pull request merged**